kafka/src/main/java/io/micronaut/configuration/kafka/ConsumerRegistry.java

```diff
@@ -86,6 +96,18 @@
      */
     void pause(@NonNull String id);
 
+    /**
+     * Pause the consumer for the given ID to consume from the given topic partitions.
+     * Note that this method will request that the consumer is paused, however
+     * does not block until the consumer is actually paused.
+     * You can use the {@link #isPaused(String, Collection<TopicPartition>)} method to
+     * establish when the consumer has actually been paused for the topic partitions.
+     *
+     * @param id The id of the consumer
+     * @param topicPartitions The topic partitions to pause consuming from
+     */
+    void pause(@NonNull String id, Collection<TopicPartition> topicPartitions);
```

Comment:
Can you add `@NonNull` or `@Nullable` to the second arg and add `@since` to the javadoc